qa/suites/upgrade/hammer-x/v0-94-6-mon-overload/v0-94-6-mon-overload.yaml

```diff
@@ -0,0 +1,63 @@
+#
+# Test the expected behavior of the code that ensures proper encoding
+# when
+#
+#    CEPH_FEATURE_OSD_HITSET_GMT
+#
+# feature is not active (prevents mon overload on upgrade to jewel)
+#
+roles:
+- - mon.a
+  - osd.0
+  - osd.1
+- - osd.2
+openstack:
+- volumes: # attached to each instance
+    count: 2
+    size: 10 # GB
+tasks:
+- print: "**** Install v0.94.6
+- install:
+    tag: v0.94.6
+- ceph:
+    fs: xfs
+- print: "*** Upgrade the target that runs osd.0 and osd.1 to -x while the target that runs osd.2 stays v0.94.6"
+- install.upgrade:
+    osd.0:
+- print: "*** Restart mon.a and osd.0 so they run jewel
+- ceph.restart:
+    daemons: [mon.a, osd.0, osd.1]
```

Comment:
The idea is to restart osd.0 and osd.1 so they run jewel, and leave osd.2 (on a separate machine) running hammer. Updated the preceding print to clarify.